igvc/launch/imu.launch

```diff
@@ -1,4 +1,21 @@
 <!-- imu.launch -->
 <launch>
-    <node name="imu" pkg="igvc" type="imu"/>
+    <node name="imu" pkg="igvc" type="imu" output="screen" >
+      <param name="yaw_offset" type="double" value="0"/>
```

Comment:
This is due to gyro drift. Not much that can be done. There does not really seem to be a pattern.